### ISSUE TITLE
Whether VNE003 is necessary

### ISSUE THREAD

**Issue by conformist-mw** [best-doctor/flake8-variables-names#14]
First of all, thank you for a great plugin!

I've also been using flake8-builtins in my regular plugin set for quite some time, it solves one specific problem for it. 

But I noticed that your plugin also detects builtins and needs to add a separate code so that flake8 does not complain.

For example:

```python
    id = Column(Integer, primary_key=True)  # noqa: A003, VNE003
```

Sure, I could just add VNE003 to ignore list, but maybe there's a way to make the plugin more specialized?

Thanks!

**Comment by Arondit** [comment]
@conformist-mw 

We're glad to see usefulness of our plugin!

About the case, we can't just remove `VNE003`, because we can't guarantee, that every user of the plugin uses also `flake8-builtins`.  So, the only way is to add `VNE003` to ignore list in the config file. Thank you for asking.

**Comment by Skorpyon** [comment]
@conformist-mw About A003, it have ignorelist:
```makefile
[flake8]
builtins-ignorelist=id
```
